### PR TITLE
Added Ruby integrations activecube

### PR DESCRIPTION
**Pull request by astudnev**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one): 

Documentation (changelog entry is not required)

Detailed description / Documentation draft:

Added Ruby integrations activecube

### CODE REVIEW

**Comment by blinkov** [comment]
@astudnev could you make the same patch for other documentation languages?

**Comment by astudnev** [comment]
Added

**Comment by qoega** [comment]
I see that activecube uses another project https://github.com/PNixx/clickhouse-activerecord.

**Comment by astudnev** [comment]
Yes, https://github.com/PNixx/clickhouse-activerecord used as ActiveRecord engine to connect to clickhouse. This one used for testing and in production configuration. I had to fork it because of some issues with building queries to https://github.com/bitquery/clickhouse-activerecord.git

File: docs/en/interfaces/third-party/integrations.md [hunk 1]
```diff
@@ -96,5 +96,11 @@ toc_title: Integrations
 -   Elixir
     -   [Ecto](https://github.com/elixir-ecto/ecto)
         -   [clickhouse\_ecto](https://github.com/appodeal/clickhouse_ecto)
+-   Ruby
+    -   [Ruby on rails](https://rubyonrails.org/)
```
Comment (blinkov) [inline docs/en/interfaces/third-party/integrations.md:100]:
```suggestion
    -   [Ruby on Rails](https://rubyonrails.org/)
```

File: docs/es/interfaces/third-party/integrations.md [hunk 2]
```diff
@@ -98,5 +98,10 @@ toc_title: "Integraci\xF3n"
 -   Elixir
     -   [Ecto](https://github.com/elixir-ecto/ecto)
         -   [Método de codificación de datos:](https://github.com/appodeal/clickhouse_ecto)
+-   Ruby
+    -   [Ruby on rails](https://rubyonrails.org/)
```
Comment (blinkov) [inline docs/es/interfaces/third-party/integrations.md:102]:
```suggestion
    -   [Ruby on Rails](https://rubyonrails.org/)
```

File: docs/fa/interfaces/third-party/integrations.md [hunk 3]
```diff
@@ -95,5 +95,10 @@ toc_title: "\u06CC\u06A9\u067E\u0627\u0631\u0686\u06AF\u06CC"
 -   اکسیر
     -   [Ecto](https://github.com/elixir-ecto/ecto)
         -   [حذف جستجو](https://github.com/appodeal/clickhouse_ecto)
-
+-   Ruby
+    -   [Ruby on rails](https://rubyonrails.org/)
```
Comment (blinkov) [inline docs/fa/interfaces/third-party/integrations.md:99]:
```suggestion
    -   [Ruby on Rails](https://rubyonrails.org/)
```

File: docs/fr/interfaces/third-party/integrations.md [hunk 4]
```diff
@@ -95,5 +95,10 @@ toc_title: "Int\xE9gration"
 -   Elixir
     -   [Ecto](https://github.com/elixir-ecto/ecto)
         -   [clickhouse\_ecto](https://github.com/appodeal/clickhouse_ecto)
+-   Ruby
+    -   [Ruby on rails](https://rubyonrails.org/)
```
Comment (blinkov) [inline docs/fr/interfaces/third-party/integrations.md:99]:
```suggestion
    -   [Ruby on Rails](https://rubyonrails.org/)
```

File: docs/ja/interfaces/third-party/integrations.md [hunk 5]
```diff
@@ -95,5 +95,10 @@ toc_title: "\u7D71\u5408"
 -   エリクサー
     -   [Ecto](https://github.com/elixir-ecto/ecto)
         -   [clickhouse\_ecto](https://github.com/appodeal/clickhouse_ecto)
+-   Ruby
+    -   [Ruby on rails](https://rubyonrails.org/)
```
Comment (blinkov) [inline docs/ja/interfaces/third-party/integrations.md:99]:
```suggestion
    -   [Ruby on Rails](https://rubyonrails.org/)
```

File: docs/ru/interfaces/third-party/integrations.md [hunk 6]
```diff
@@ -91,5 +91,10 @@
 -   Elixir
     -   [Ecto](https://github.com/elixir-ecto/ecto)
     -   [clickhouse\_ecto](https://github.com/appodeal/clickhouse_ecto)
-
+-   Ruby
+    -   [Ruby on rails](https://rubyonrails.org/)
```
Comment (blinkov) [inline docs/ru/interfaces/third-party/integrations.md:95]:
```suggestion
    -   [Ruby on Rails](https://rubyonrails.org/)
```

File: docs/tr/interfaces/third-party/integrations.md [hunk 7]
```diff
@@ -95,5 +95,10 @@ toc_title: Entegrasyonlar
 -   İksir
     -   [Ecto](https://github.com/elixir-ecto/ecto)
         -   [clickhouse\_ecto](https://github.com/appodeal/clickhouse_ecto)
+-   Ruby
+    -   [Ruby on rails](https://rubyonrails.org/)
```
Comment (blinkov) [inline docs/tr/interfaces/third-party/integrations.md:99]:
```suggestion
    -   [Ruby on Rails](https://rubyonrails.org/)
```

File: docs/zh/interfaces/third-party/integrations.md [hunk 8]
```diff
@@ -89,5 +89,10 @@
 -   仙丹
     -   [Ecto](https://github.com/elixir-ecto/ecto)
     -   [clickhouse\_ecto](https://github.com/appodeal/clickhouse_ecto)
+-   Ruby
+    -   [Ruby on rails](https://rubyonrails.org/)
```
Comment (blinkov) [inline docs/zh/interfaces/third-party/integrations.md:93]:
```suggestion
    -   [Ruby on Rails](https://rubyonrails.org/)
```

**Comment by qoega** [comment]
1) Probably  https://github.com/PNixx/clickhouse-activerecord  should be added in this list as well.
2) When I opened your link, I could find ClickHouse mention only in this tests section
https://github.com/bitquery/activecube#rspec-tests
3)And nothing about ClickHouse in second one
https://github.com/bitquery/activecube-graphql/search?q=clickhouse&unscoped_q=clickhouse
4) Even if ClickHouse is supported it is hard to see it on the page. When I open https://github.com/PNixx/clickhouse-activerecord it is obvious

**Comment by qoega** [comment]
@astudnev Can you approve changes from Ivan and me? Or add rights to change PR.
And we will merge it

**Comment by alexey-milovidov** [comment]
@qoega ping.

**Comment by alexey-milovidov** [comment]
@qoega has failed integrating this PR.
I will do.

**Pull request merged**